bindings/jupyter-modules/jupyter-notebook-search/src/index.js

```diff
@@ -1,3 +1,26 @@
-import React from 'react';
+import React, {useEffect, useRef} from 'react';
+import * as d3 from 'd3';
+export default props => {
+  const svgRef = useRef(null);
+  const {data} = props;
+  const cells = (data && data.notebooks) || [];
+  useEffect(() => {
+    const sel = d3
+      .select(svgRef.current)
```

Comment:
we might need some inline comments to help Tim onboard React.

Reply:
added the inline comments